upgrade/upgrade.go

```diff
@@ -230,6 +207,31 @@ func doAPIVersionUpgrade(namespace string, config *restclient.Config, lhClient *
 	return nil
 }
 
+func canUpgrade(config *restclient.Config, namespace string) (bool, error) {
+	lhClient, err := lhclientset.NewForConfig(config)
+	if err != nil {
+		return false, errors.Wrap(err, "unable to get clientset for v1beta1")
+	}
+
+	scheme := runtime.NewScheme()
+	if err := longhorn.SchemeBuilder.AddToScheme(scheme); err != nil {
+		return false, errors.Wrap(err, "unable to create scheme for v1beta1")
+	}
+
+	setting, err := lhClient.LonghornV1beta2().Settings(namespace).Get(context.TODO(), string(types.SettingNameDefaultEngineImage), metav1.GetOptions{})
```

Comment:
Shouldn't this be `LonghornV1beta1`?

Reply:
Yes, LonghornV1beta1 is better although LonghornV1beta1 and LonghornV1beta2 for setting are equivalent.